Resources/public/js/views/universaldiscovery/ez-universaldiscoveryfinderview.js

```diff
@@ -89,6 +89,11 @@ YUI.add('ez-universaldiscoveryfinderview', function (Y) {
          * @param {eventFacade} e.data the node data
          */
         selectContent: function (e) {
+            //fix for firefox 60.0+ selection start on click
```

Comment:
If only FF is affected you should check if the user uses FF.

Reply:
Done